components/03-modules/active-filters/active-filters.hbs

```diff
@@ -0,0 +1,21 @@
+<ul class="active-filters list">
+    {{# each filters }}
+        <li class="active-filters__item list__item">
+            <button type="button"
+                    class="active-filters__remove"
+                    aria-label="{{ariaLabel}}"
+            >
+                <svg class="active-filters__remove-icon">
+                    <use xlink:href="{{ svg iconId }}"></use>
+                </svg>
+            </button>
+            <span class="active-filters__label">{{label}}&nbsp;:&nbsp;</span>
+            <span class="active-filters__value">{{value}}</span>
+        </li>
+    {{/each}}
+    <a href="{{clearHref}}"
+       class="active-filters__clear"
```

Comment:
shouldn't we add here also `{{clearClass}}` ?

Reply:
Done.